CalcualtorLogic.js

```diff
@@ -31,6 +31,9 @@ class Calculator {
     render() {
         const wrapperDiv = document.createElement('div');
         wrapperDiv.classList.add("wrapper");
+        // console.log(wrapperDiv.outerHTML);
+        const resultDiv = document.createElement('div');
+        resultDiv.classList.add("result");
```

Comment:
what about the id attribute?
the result div has `id` also

   <div id="result" class="result">0</div>

`resultDiv.id = "result"`
can you add that also?

Reply:
ok

Reply:
Now you can append  **resultDiv** to **wrapperDiv**;

You can use the `appendChild` method which available on *wrapperDiv*

Example:
```ts
// Create a new paragraph element, and append it to the end of the document body
let p = document.createElement("p");
document.body.appendChild(p);
```